src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java

```diff
@@ -1,16 +1,24 @@
 package com.fabriccommunity.spookytime.common;
 
 import com.fabriccommunity.spookytime.SpookyTime;
-import net.minecraft.block.Block;
+import com.fabriccommunity.spookytime.common.block.DeceasedGrassBlock;
+import net.fabricmc.fabric.api.block.FabricBlockSettings;
+import net.minecraft.block.*;
```

Comment:
Not sure whether using a star import on the block package is a good idea.

Reply:
This actually wasn’t intentional, IDEA seems to have done it itself. I’ll change the imports when I can.